### DISCUSSION TITLE
Access to ghe student pack

### DISCUSSION THREAD

**Discussion by Muzorava** [community/community#147036] [GitHub Education]
### Select Topic Area

Question

### Body

<img width="917" alt="Screenshot 2024-12-13 193929" src="https://github.com/user-attachments/assets/611c2e99-e1b7-43e1-9eab-da711ee2f9d4" />
Hello! Anyone to help me on this, i got the ghe student pack but im failing to have access to them.

**Comment by shinybrightstar** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Muzorava ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Code Search and Navigation category is a place for our community to discuss and provide feedback on GitHub search and navigation. GitHub code search lets you rapidly search, navigate and understand your code, your team's code, and the code of the open source community.


I've gone ahead and moved this to the GitHub Education category for you. Good luck!
